src/components/Avatar.js

```diff
@@ -106,9 +107,9 @@ function Avatar(props) {
             ) : (
                 <View style={[iconStyle, StyleUtils.getAvatarBorderStyle(props.size, props.type), ...props.iconAdditionalStyles]}>
                     <Image
-                        source={{uri: props.source}}
+                        source={{uri: props.source, params: {cacheBuster}}}
```

Comment:
`params` is a part of the `uri`, instead of sending both `uri` and `params` let's update the `uri` here (in Avatar) and send the updated `uri`.